ups/meas_extensions_gaap.table

```diff
@@ -0,0 +1,9 @@
+setupRequired(meas_algorithms)
+setupRequired(meas_extensions_gaap)
+setupRequired(afw)
+setupRequired(geom)
+setupRequired(meas_base)
+setupRequired(utils)
+setupRequired(daf_base)
```

Comment:
This list needs at least `ip_diffim` as well.  Please check the others, too, and include only packages whose modules you have directly imported.  It's unfortunately hard to keep these in sync with what's actually used, but we should start off with a careful one.